editor.planx.uk/src/@planx/components/List/Public/index.tsx

```diff
@@ -181,7 +181,7 @@ const Root = () => {
               sx={{ width: "100%" }}
               data-testid="list-add-button"
             >
-              + Add a new {schema.type.toLowerCase()} description
+              + Add another {schema.type.toLowerCase()} description
```

Comment:
One of the GLA schemas has `type: "New built residentail unit"`, so "another" prevents "new new"